learners/trial-runs.md

```diff
@@ -2,5 +2,124 @@
 title: "Lesson Trial Runs"
 ---
 
-TODO
+[Teaching a lesson for the first time is an important intermediate step in the lesson development process](../episodes/17-operations.md).
+During the break between the first and second parts of this training,
+we ask trainees to teach part of the lesson they have been designing to a real audience.
+The purpose of this task is to gather feedback at an early stage of the development process,
+and to reflect on how well the tested lesson content meets the objectives that we defined for it.
+
+On this page, we provide details of the task and 
+some guidance on what feedback/information to collect while testing out the lesson content.
+
+
+## What should I teach?
+
+During the first stages of the training, you will have completed 
+the early stages of design and development of a chapter of your new lesson.
```

Comment:
Are we moving away from using "episode"?  Either way, maybe we want to be consistent with what we call these parts of a lesson.

Reply:
you are right, and thanks for catching this. The plan is definitely to move away from "episode" towards "chapter" but we have not made the transition yet elsewhere. I guess it is better to call everything an episode for now, if for no other reason than that is the name of the directory in a lesson repository. I will work up a separate PR to make the terminology consistent throughout this lesson, because I expect I have let "chapter" slip through elsewhere too